### PR TITLE
feat: view follow suggestions

### PR DESCRIPTION
**Pull request by technicat**
TootClient.getSuggestions() performs this query

https://docs.joinmastodon.org/methods/suggestions/

### CODE REVIEW

File: Sources/TootSDK/Models/Suggestion.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    /// The reason this account is being suggested.
+    public var source: Source
+    /// The account being recommended to follow.
+    public var account: Account
+
+    public enum Source: String, Codable, Sendable {
+        /// This account was manually recommended by your administration team
+        case staff
+        /// You have interacted with this account previously
+        case past_interactions
```
Comment (Tunous) [inline Sources/TootSDK/Models/Suggestion.swift:26]:
```suggestion
        case pastInteractions = "past_interactions"
```

File: Sources/TootSDK/Models/Suggestion.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        /// This account has many reblogs, favourites, and active local followers within the last 30 days
+        case global
+    }
+
+    enum CodingKeys: String, CodingKey {
+        case source
+        case account
+    }
```
Comment (Tunous) [inline Sources/TootSDK/Models/Suggestion.swift:34]: Not needed, it's the same as automatically generated one.

```suggestion
```

File: Sources/TootSDK/Models/Suggestion.swift (hunk 1 above)
Comment (davidgarywood) [reply Sources/TootSDK/Models/Suggestion.swift:26]: Do we need this? (The decoder is set to handle camel case coming in)

Comment (Tunous) [reply Sources/TootSDK/Models/Suggestion.swift:26]: You're right, fixed my suggestion.

Comment (davidgarywood) [reply Sources/TootSDK/Models/Suggestion.swift:26]: @technicat just to say this is a great addition - if we can update this case to snake case, then I think this is good to go! 🙏

Comment (technicat) [reply Sources/TootSDK/Models/Suggestion.swift]: Went with the first version of @Tunous suggestion, got a decoder error with the second. Here's how it looks now on universeodon.com

IMG_2387

**Review by davidgarywood — APPROVED** [review]
🙌

File: Sources/TootSDK/Models/Suggestion.swift (hunk 1 above)
Comment (Tunous) [reply Sources/TootSDK/Models/Suggestion.swift]: Ah, camel case conversion works for fields not enum values. That's why my first suggestion worked :)

**Pull request merged**